src/things.ts

```diff
@@ -111,6 +111,29 @@ export const Things = {
 		}
 		return Labels.Localize(thing.label);
 	},
+	DrawWiki: (thing: Thing): HTMLElement => {
+		const wiki = document.createFullElement('div', {class: 'wiki'});
+		const link = document.createFullElement('a', {href: Router.GetWikiUrl(thing), target: '_blank'});
+		const header = document.createFullElement('span', {class: 'header'});
+		const footer = document.createFullElement('span', {class: 'footer'});
+		const highlightText = document.createFullElement('strong');
```

Comment:
Could you please use snake case as for the other variables?

Reply:
Good catch, will change.